Detections/SecurityEvent/SecurityEventLogCleared.yaml

```diff
@@ -1,7 +1,8 @@
 ﻿id: 80da0a8f-cfe1-4cd0-a895-8bc1771a720e
 name: Security Event log cleared
 description: |
-  'Checks for event id 1102 which indicates the security event log was cleared.'
+  'Checks for event id 1102 which indicates the security event log was cleared. 
+It uses Event Source Name "Microsoft-Windows-Eventlog" to avoid generating false positives from other sources, like AD FS servers for instance.'
```

Comment:
Something I did not notice before, we need 2 spaces here, this breaks YAML, please add the additional space in front of It.  You can verify format with any online YAML validator, I often use - https://codebeautify.org/yaml-validator/

Reply:
Thank you for that! Validated :)